rotkehlchen/history/price.py

```diff
@@ -167,7 +167,7 @@ def query_historical_price(
             from_asset: The ticker symbol of the asset for which we want to know
                         the price.
             to_asset: The ticker symbol of the asset against which we want to
-                      know the price.
+                        know the price.
             timestamp: The timestamp at which to query the price
```

Comment:
why this change?

Reply:
aligning with spaces of 4, it had a block of two spaces, was showing red on vscode.